src/style/star.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars__star:nth-child(n+5) {
```

Comment:
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 3rd item, you should have all the modifiers for different number of stars